### ISSUE TITLE
Not working in Inbox

### ISSUE THREAD

**Issue by selrond** [adam-p/markdown-here#327]
Markdown-marked-up text does not react to context menu toggle, nor keyboard shortcut... Same for quick compose, as well as in reply mode...

**Comment by tatianagrange** [comment]
It is the same for me. In quick answer and in reply mode.

**Comment by thaldyron** [comment]
Same problem for me.

**Comment by brendanmatkin** [comment]
Can confirm this.. using chrome 53.0.2785.143 on OS X 10.11.6

**Comment by alexozer** [comment]
Isn't working for me either. I sometimes get a popup asking to confirm unrendering the changes, which produces glitchy behavior.

**Comment by fredley** [comment]
Having installed on Chrome, the help page popped open half way through writing an email (a good 100+ chars in, at least 20 seconds after starting typing). There is no M button on the toolbar, keyboard shortcuts and context menu do nothing.

**Comment by timvdalen** [comment]
Came here from the HN thread. The extension works fine for me in Inbox.

![screenshot 2017-11-08 at 14 18 55](https://user-images.githubusercontent.com/1038428/32551147-d7a518d2-c48f-11e7-9085-a65dc8180fa5.png)

**Comment by alexozer** [comment]
Tried this again, seemed to realize the paradigm is "edit in plain-text, convert, unconvert if need more edits". Actually seems to work fine now for me!

**Comment by tomasz-wiszkowski** [comment]
I don't think this is about the paradigm. In fact, I'm suffering from that same problem as OP.

I have the markdown toggle button enabled, but it doesn't do anything.
Keyboard shortcuts don't work.

I can open markdown options page and successfully use my shortcut (or button) to toggle between rendered and plain text modes. Inbox and gmail don't work for me at all.

It used to work just a few months ago; I used it in December if i'm not mistaken and it worked. Right now it no longer does anything

**Comment by ChuliangXiao** [comment]
It didn't work for me at first. 
After I closed  and relaunched Chrome, it worked.

**Comment by lrdegeest** [comment]
Just tried this extension on Chrome. Does not work.
